ci/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+#
+# Build snowflake-sqlalchemy
+set -o pipefail
+
+PYTHON="python3.7"
```

Comment:
> this is to use py3.7 to build the universal wheel as we only support py>= 3.7 now, right?

No, it's just the oldest Python version still supported by the open-source community

Feel free to create a Jira and submit a PR for this.
There was no reason to discontinue support for older versions of Python before (although; I think we added some f-strings and so that would mean that it doesn't work with 3.5 anymore.

Now that development will ramp up soon I have no objection against restricting supported versions and modernizing the code-base.